spec/BowlingSpec.js

```diff
@@ -0,0 +1,64 @@
+describe('Bowling Scorecard', function() {
+  var scorecard, frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  describe('bowls a single ball and –', function() {
+    describe('keeps track of the frame’s score, which:', function() {
+      it('is initally 0', function() {
+        expect(frame.frameScore).toEqual(0);
+      });
+
+      it('(potentially) increases as balls are bowled', function() {
+        frame.bowl();
+        expect(frame.frameScore >= 0).toBeTruthy();
+      });
+
+      it('does not exceed the initial number of skittles', function() {
+        initialNumberOfSkittles = frame.skittlesRemaining;
+        frame.bowl();
```

Comment:
Might have been tempted to stub the response to `bowl()` to be a positive integer - you run the risk of getting a 'false positive' result for this test if, say, `bowl()` returns `null`